wordpress-mobile/WordPress-Android
# Issue/9459 update row type include percentage bar in stats

Fixes #9459 - update row type include percentage bar in stats

This PR makes adjustments to the following stats sections:
Tags and Categories on Insights tab
Posts and Pages on time based tabs (i,e, Days, Weeks, Months and Years)
Authors on time based tabs

The adjustment is to add a horizontal bar within each element to represent the proportion of views for that element relative to the top most element.

For example, the top most element with 51 views will have a bar that goes right across (100%).  The next element with 22 views will have a bar that goes approximately 43% of the way across.

For further clarity, please see the below screenshot.
![Screenshot_1556885320](https://user-images.githubusercontent.com/48055157/57136451-d3fabd80-6da4-11e9-9812-2c6c6fbf52f6.png)

### To Test

Login with any account that has access to sites with sufficient stats.

- [ ] View the Tags and Categories section on the insights tab.  A bar should should be visible for each element demonstrating the proportion of the overall views.
- [ ] Tap View more (where available) to see the details list showing the same bar for all elements in the list.
- [ ] View the Posts and Pages section on any of the time based tabs (e.g. Days).  A similar bar should show.
- [ ] Check the bar shows on the View more for the Posts and Pages section as well.
- [ ] View the Authors section on a time based tab.  A similar bar should show.
- [ ] Expand any of the authors elements using the chevron to the right.  The bar should _only_ show per author.  Not on individual author posts.  See screenshot above.
- [ ] Check the bar shows in a similar fashion on the View more screen for Authors where available.
- [ ] Check the bar is not showing on any other sections.

Update release notes:


Also see the before and after images below.

### Before
![Screenshot_1556813865](https://user-images.githubusercontent.com/48055157/57091584-79f4ec00-6d01-11e9-99b7-8a299be301ae.png)
![Screenshot_1556813873](https://user-images.githubusercontent.com/48055157/57091596-7e210980-6d01-11e9-9198-7120358de53e.png)
![Screenshot_1556813886](https://user-images.githubusercontent.com/48055157/57091600-80836380-6d01-11e9-8fea-f781e84ae5f9.png)
![Screenshot_1556813898](https://user-images.githubusercontent.com/48055157/57091608-837e5400-6d01-11e9-884a-a09aee4957b6.png)

### After
![Screenshot_1556885288](https://user-images.githubusercontent.com/48055157/57136456-d9580800-6da4-11e9-9cce-d3451cb561ba.png)
![Screenshot_1556885320](https://user-images.githubusercontent.com/48055157/57136463-dceb8f00-6da4-11e9-945e-8de579418b61.png)
![Screenshot_1556885340](https://user-images.githubusercontent.com/48055157/57136465-e07f1600-6da4-11e9-8022-b57e1d9e6216.png)
